packages/terra-search-field/src/SearchField.jsx

```diff
@@ -107,15 +111,15 @@ class SearchField extends React.Component {
     this.handleTextChange = this.handleTextChange.bind(this);
     this.handleSearch = this.handleSearch.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.handleInput = this.handleInput.bind(this);
     this.setInputRef = this.setInputRef.bind(this);
-    this.updateSearchText = this.updateSearchText.bind(this);
```

Comment:
I think we still want this to be bound

Reply:
Fixed https://github.com/cerner/terra-core/pull/2800/commits/7f1b23a3ff00608c4dcf47ceae24a47d2b94db4d#diff-105fddc0e7a42eb008cb5c09faad5222R116